hathor/p2p/protocol.py

```diff
@@ -92,11 +92,12 @@ class WarningFlags(str, Enum):
     capabilities: set[str]  # capabilities received from the peer in HelloState
 
     def __init__(self, network: str, my_peer: PeerId, p2p_manager: 'ConnectionsManager',
-                 *, use_ssl: bool, inbound: bool) -> None:
+                 *, use_ssl: bool, inbound: bool, remote_address: 'IAddress') -> None:
```

Comment:
Is it possible to use `remote_address: IAddress` instead of using a string annotation?

Reply:
Done!